frontend/src/store/modules/oauth.ts

```diff
@@ -16,14 +16,23 @@ const actions = {
     {
       vcsId,
       code,
+      oauthConfig,
     }: {
       vcsId: VCSId;
```

Comment:
nit: The VCSId here would be undefined when we first set up the VCS for the workspace, maybe it is better use `vcsId: VCSId | underfined` instead of `vcsId: VCSId`

Reply:
I took the suggestion from @tianzhou with `vcsId? : VCSId`, is this the same thing? (frontend noob 😁 )